src/main/scala/Plugin.scala

```diff
@@ -1,460 +0,0 @@
-      case (Some(user), Some(pass)) =>
-        (user, pass)
-      case _ =>
-        val propsCredentials = for (name <- sys.props.get("sona.user"); pass <- sys.props.get("sona.pass")) yield (name, pass)
-        propsCredentials match {
-          case Some((name, pass)) => (name, pass)
-          case _ =>
-            val envCredentials = for (name <- sys.env.get("SONA_USER"); pass <- sys.env.get("SONA_PASS")) yield (name, pass)
-            envCredentials.getOrElse {
-              val name = Prompt("Enter sonatype username").getOrElse {
-                sys.error("sonatype username required")
-              }
-              val pass = Prompt.descretely("Enter sonatype password").getOrElse {
-                sys.error("sonatype password is required")
-              }
-              (name, pass)
-            }
-        }
-    }
-  }
-
-  // todo: generalize this for both bintray & sonatype credential prompts
-  private def requestCredentials(
-    defaultName: Option[String] = None,
-    defaultKey: Option[String] = None): (String, String) = {
-    val name = Prompt("Enter bintray username%s" format(
-      defaultName.map(" (%s)".format(_)).getOrElse(""))).orElse(defaultName).getOrElse {
-      sys.error("bintray username required")
-    }
-    val pass = Prompt.descretely("Enter bintray API key %s" format(
-      defaultKey.map(_ => "(use current)").getOrElse("(under https://bintray.com/profile/edit)")))
-        .orElse(defaultKey).getOrElse {
-          sys.error("bintray API key required")
-        }
-    (name, pass)
-  }
-
-  private def saveBintrayCredentials(to: File)(creds: (String, String)) = {
-    println(s"saving credentials to $to")
-    val (name, pass) = creds
-    BintrayCredentials.writeBintray(name, pass, to)
-    println("reload project for sbt setting `publishTo` to take effect")
-  }
-
-  /** tries to resolve bintray credentials from sbt's credentials key, then bintray's credentialsFile.
-   *  if prompt is true, we enter an interactive mode to collect them from the user */
-  private def ensuredCredentials(
-    credsFile: File, prompt: Boolean = true): Option[BintrayCredentials] =
-        BintrayCredentials.read(credsFile).fold(sys.error(_), _ match {
-          case None =>
-            if (prompt) {
-              println("bintray-sbt requires your bintray credentials.")
-              saveBintrayCredentials(credsFile)(requestCredentials())
-              ensuredCredentials(credsFile, prompt)
-            } else {
-              println(s"Missing bintray credentials $credsFile. Some bintray features depend on this.")
-              None
-            }
-          case creds => creds
-        })
-
-  def bintrayPublishSettings: Seq[Setting[_]] = bintrayCommonSettings ++ Seq(
-    credentialsFile in bintray in Global := Path.userHome / ".bintray" / ".credentials",
-    name in bintray := moduleName.value,
-    bintrayOrganization in bintray in Global := { if (sbtPlugin.value) Some("sbt") else None },
```

Comment:
I'm trying to do a release of these changes and noticed there are a handful of unexpected gotchas a missed when going over this and I'm not sure if they were intentional or not. 

One of them changed this to deftault to None and repository in bintray to "sbt-plugins"

I'm going to revert that to the previous values but let me know if this was an intentional change